### DISCUSSION TITLE
Server not starting up after latest pull

### DISCUSSION THREAD

**Discussion by iturdzonyou** [itzg/docker-minecraft-server#2818] [General]
So I had the server running for a a few months maybe about a year or so ago. I stopped it since we all stopped playing, and now we're looking to start back up. Upon loading everything back up, I was able to load in.

I exited and did _docker-compose pull atm8_ , to get the latest version of the image and here we are. I got this error and do not know where to go from here. 

I went as far as deleting the entire container and recreated it from scratch and still no success.

[ATM8 Error.txt](https://github.com/itzg/docker-minecraft-server/files/15245399/ATM8.Error.txt)

**Comment by itzg** [comment]
I can't view the logs on my phone, but I'm guessing you need to pin the image tag to Java 17

https://docker-minecraft-server.readthedocs.io/en/latest/versions/java/

**Reply by iturdzonyou** [reply]
Okay that fixed that part. Now i'm getting this error

atm8            | Caused by: io.netty.resolver.dns.DnsNameResolverTimeoutException: [4139: /100.100.100.100:53] DefaultDnsQuestion(raw.githubusercontent.com. IN A) query '4139' via UDP timed out after 5000 milliseconds (no stack trace available)
atm8            | [init] ERROR failed to auto-install CurseForge modpack
atm8 exited with code 1
atm8            | [init] Running as uid=1000 gid=1000 with /data as 'drwxr-xr-x 19 1000 1000 4096 May  8 23:40 /data'
atm8            | [mc-image-helper] 15:30:27.159 ERROR : 'install-curseforge' command failed. Version is 1.38.14
atm8            | reactor.core.Exceptions$ReactiveException: io.netty.resolver.dns.DnsResolveContext$SearchDomainUnknownHostException: Failed to resolve 'raw.githubusercontent.com' [A(1)] and search domain query for configured domains failed as well: [tail14215.ts.net]
atm8            |       at reactor.core.Exceptions.propagate(Exceptions.java:408)
atm8            |       at reactor.core.publisher.BlockingSingleSubscriber.blockingGet(BlockingSingleSubscriber.java:101)
atm8            |       at reactor.core.publisher.Mono.block(Mono.java:1712)
atm8            |       at me.itzg.helpers.http.ObjectFetchBuilder.execute(ObjectFetchBuilder.java:44)
atm8            |       at me.itzg.helpers.curseforge.InstallCurseForgeCommand.loadExcludeIncludesFile(InstallCurseForgeCommand.java:336)
atm8            |       at me.itzg.helpers.curseforge.InstallCurseForgeCommand.loadExcludeIncludes(InstallCurseForgeCommand.java:279)
atm8            |       at me.itzg.helpers.curseforge.InstallCurseForgeCommand.call(InstallCurseForgeCommand.java:198)
atm8            |       at me.itzg.helpers.curseforge.InstallCurseForgeCommand.call(InstallCurseForgeCommand.java:30)
atm8            |       at picocli.CommandLine.executeUserObject(CommandLine.java:2041)
atm8            |       at picocli.CommandLine.access$1500(CommandLine.java:148)
atm8            |       at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
atm8            |       at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
atm8            |       at picocli.CommandLine$RunLast.handle(CommandLine.java:2415)
atm8            |       at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
atm8            |       at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
atm8            |       at picocli.CommandLine.execute(CommandLine.java:2170)
atm8            |       at me.itzg.helpers.McImageHelper.main(McImageHelper.java:159)
atm8            |       Suppressed: java.lang.Exception: #block terminated with an error
atm8            |               at reactor.core.publisher.BlockingSingleSubscriber.blockingGet(BlockingSingleSubscriber.java:103)
atm8            |               ... 15 common frames omitted
atm8            | Caused by: io.netty.resolver.dns.DnsResolveContext$SearchDomainUnknownHostException: Failed to resolve 'raw.githubusercontent.com' [A(1)] and search domain query for configured domains failed as well: [tail14215.ts.net]
atm8            |       at io.netty.resolver.dns.DnsResolveContext.finishResolve(DnsResolveContext.java:1151)
atm8            |       at io.netty.resolver.dns.DnsResolveContext.tryToFinishResolve(DnsResolveContext.java:1098)
atm8            |       at io.netty.resolver.dns.DnsResolveContext.query(DnsResolveContext.java:457)
atm8            |       at io.netty.resolver.dns.DnsResolveContext.access$700(DnsResolveContext.java:69)
atm8            |       at io.netty.resolver.dns.DnsResolveContext$2.operationComplete(DnsResolveContext.java:526)
atm8            |       at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:590)
atm8            |       at io.netty.util.concurrent.DefaultPromise.notifyListeners0(DefaultPromise.java:583)
atm8            |       at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:559)
atm8            |       at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:492)
atm8            |       at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:636)
atm8            |       at io.netty.util.concurrent.DefaultPromise.setFailure0(DefaultPromise.java:629)
atm8            |       at io.netty.util.concurrent.DefaultPromise.tryFailure(DefaultPromise.java:118)
atm8            |       at io.netty.resolver.dns.DnsQueryContext.finishFailure(DnsQueryContext.java:380)
atm8            |       at io.netty.resolver.dns.DnsQueryContext$5.run(DnsQueryContext.java:315)
atm8            |       at io.netty.util.concurrent.PromiseTask.runTask(PromiseTask.java:98)
atm8            |       at io.netty.util.concurrent.ScheduledFutureTask.run(ScheduledFutureTask.java:153)
atm8            |       at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:173)
atm8            |       at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:166)
atm8            |       at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470)
atm8            |       at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:416)
atm8            |       at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
atm8            |       at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
atm8            |       at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
atm8            |       at java.base/java.lang.Thread.run(Unknown Source)
atm8            | Caused by: io.netty.resolver.dns.DnsNameResolverTimeoutException: [5412: /100.100.100.100:53] DefaultDnsQuestion(raw.githubusercontent.com. IN A) query '5412' via UDP timed out after 5000 milliseconds (no stack trace available)
atm8            | [init] ERROR failed to auto-install CurseForge modpack








Basically - atm8            | [init] ERROR failed to auto-install CurseForge modpack

**Reply by itzg** [reply]
That looks like a DNS issue you'll need to solve on your system

```
DnsNameResolverTimeoutException: [4139: /100.100.100.100:53] 
```

**Reply by iturdzonyou** [reply]
Alright fixed the DNS issue and it looks like its starting up correctly. Hopes and prayers that this is the last issue!
